Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/MainWindow.h

```diff
@@ -128,7 +128,7 @@ namespace EMStudio
 
         GUIOptions& GetOptions()                                                { return m_options; }
 
-        void Reset(bool clearActors = true, bool clearMotionSets = true, bool clearMotions = true, bool clearAnimGraphs = true, MCore::CommandGroup* commandGroup = nullptr);
+        void Reset(bool clearActors = true, bool clearMotionSets = true, bool clearMotions = true, bool clearAnimGraphs = true, MCore::CommandGroup* commandGroup = nullptr, bool addDefaultMotionSet = true);
```

Comment:
nit: would be better if we have a ResetSetting structure than passing all of the option in the function param